### PR TITLE
Reporting remake

### PR DESCRIPTION
**Pull request by Redmonkeycloud**

### CODE REVIEW

**Comment by derevirn** [comment]
@Redmonkeycloud Has the compression library error been fixed?

File: main.gms [hunk 1]
```diff
@@ -126,6 +126,8 @@ $elseif.loadData %DevMode% == 1 $call "RScript ./loadMadratData.R DevMode=1"
 $elseif.loadData %DevMode% == 2 $call "RScript ./loadMadratData.R DevMode=2"
 $endif.loadData
 $endif.genInp
+file fStat /'modelstat.txt'/;
```
Comment (derevirn) [inline main.gms:129]: Add `modelstat.txt` to `.gitignore` so Git ignores it when created in the root folder.

Comment (Redmonkeycloud) [reply main.gms:129]: Yes, the issue has been resolved.

Comment (Redmonkeycloud) [reply main.gms:129]: .gitignore  updated

**Review by derevirn — APPROVED** [review]

**Review by derevirn — APPROVED** [review]

**Pull request merged**